Gooey.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name                  = 'Gooey'
-  s.version               = '1.0.1'
+  s.version               = '1.0.2'
```

Comment:
I think '1.1.0' right?

Reply:
Hmm, seems like just a bug fix / patch change to me? I'm fine either way - but this isn't _adding_ functionality, it's making intended functionality work correctly.

Reply:
It did add to the public definitions of all NSObjects (in a backwards-compatible manner) but for the sake of avoiding small minor versions that seems reasonable